src/ChannelServer/World/Entities/Entity.cs

```diff
@@ -52,6 +56,11 @@ public interface IEntity
 	/// </summary>
 	public interface ICombatEntity : IEntity
 	{
+		/// <summary>
```

Comment:
We have the Components property to access components, so we don't need to put them into the interfaces like this.

Reply:
Removed and now accessing it through components.